components/director/Makefile

```diff
@@ -8,3 +8,15 @@ include $(SCRIPTS_DIR)/generic_make_go.mk
 VERIFY_IGNORE := /vendor\|/automock\|/testdata
 
 verify:: gqlgen check-gqlgen
+
+errcheck-local:
+	 GO111MODULE="on" errcheck -blank -asserts -ignoregenerated ./...
+
+test-local:
+	GO111MODULE=on go test ./...
+
+resolve-local: ;
+
+ensure-local: ;
+
+dep-status-local: ;
```

Comment:
`go mod graph`? Or just echo a message that this project uses Go modules and this step is skipped. Eventually the target will be removed, when we move all components to Go modules.